lib/autobuild/importer.rb

```diff
@@ -201,6 +201,12 @@ def retry_count=(count)
         @options[:retry_count] = Integer(count)
     end
 
+    def fingerprint(package)
+        #each importer type should implement its own 
+        Autoproj.warn "Fingerprint has not been implemented for this type of packages, results should be discarded"
```

Comment:
Rather return nil, and make sure that the `autoproj versions` CLI warns if `fingerprint` is nil.

Reply:
Also, document the method

Reply:
I added return nil, but I don't know if it would be better to raise an exception.